monai/handlers/segmentation_saver.py

```diff
@@ -43,15 +43,19 @@ def __init__(
             output_postfix (str): a string appended to all output file names.
             output_ext (str): output file extension name.
             resample (bool): whether to resample before saving the data array.
-            interp_order (int): the order of the spline interpolation, default is 0.
+            interp_order (int or str):
+                The interpolation mode. This option is used when `resample = True`.
+                When saving NIfTI files, the options are "nearest", "bilinear". Defaults to "bilinear".
+                See also: https://pytorch.org/docs/stable/nn.functional.html#grid-sample
+                When saving PNG files, this sets the order of the spline interpolation.
```

Comment:
Can we also use PyTorch Tensor API to resize image shape for PNGSaver?
Then we can use same args here.

Reply:
yes I changed it in #511 it's using a different pytorch method, couldn't use the same args easily. might need another iteration to initialise the niftisaver/pngsaver outside of segmentationsaver, and have a saver interface. it's not an urgent task for now

Reply:
OK, sounds good.